src/Raven.Server/Documents/Expiration/ExpirationStorage.cs

```diff
@@ -113,7 +108,7 @@ public record ExpiredDocumentsOptions
                     return null;
                 }
 
-                var expired = new Dictionary<Slice, List<(Slice LowerId, string Id)>>();
+                var expired = new Queue<ExpiredDocumentInfo>();
```

Comment:
Can you say something more about this change?

Reply:
We need to able to execute only part of the found expired/to refresh documents.
It's much easier to do it with a queue.